docs/modules/getting-started/pages/install-hazelcast.adoc

```diff
@@ -20,7 +20,7 @@ You must use one of the following installation methods if you want to install th
 * <<Using the Binary>>
 * <<Using Java>>
 
-To install a slim distribution, append `-slim` to the version number in the commands shown in the following sections; for example, to install the slim distribution of {full-version}, use `{full-version}-slim`.
+To install a slim distribution, append `-slim` to the version number in the commands shown in the following sections; for example, to install the slim distribution of {last-oss-version}, use `{last-oss-version}-slim`.
```

Comment:
```suggestion
To install a slim distribution, append `-slim` to the version number in the commands shown in the following sections; for example, to install the slim distribution of {os-version}, use `{os-version}-slim`.
```
If we go with suggested variables we need to change refs as per updated naming ^